server/src/routes/spotify.js

```diff
@@ -27,32 +24,76 @@ router.get("/hello", (req, res) => {
   });
 });
 
+// refresh access token path
```

Comment:
<img width="1519" alt="Screen Shot 2021-07-11 at 5 07 03 PM" src="https://user-images.githubusercontent.com/6634974/125209928-6b53d900-e26a-11eb-94be-40fe18dbf55c.png">

`/spotify/login` does not work for me here.  
I used the .env file from our discord. Did anything change?

Reply:
Yup, so you now want `.env.local` under client, I think I noted this in the desc.
So now, you need `client/.env.local`

```
NEXT_PUBLIC_SERVER_URL=http://localhost:9002
NEXT_PUBLIC_AUTH_URL=https://accounts.spotify.com/authorize?client_id=<client-id-here>&response_type=code&redirect_uri=http://localhost:9002/spotify/login&scope=streaming%20user-read-email%20user-read-private%20user-library-read%20user-library-modify%20user-read-playback-state%20user-modify-playback-state%20playlist-read-private
```